plugins/field-colour/README.md

```diff
@@ -30,11 +32,15 @@ This field accepts up to 4 parameters:
 - "columns" to specify the number of columns the colour dropdown should have.
   Defaults to 7.
 
-### JavaScript
+If you want to use only the field, you must install it by calling `registerFieldColour` before instantiating your blocks. If another field is registered under the same name, this field will overwrite it.
+
+#### JavaScript
 
 ```js
 import * as Blockly from 'blockly';
 import {FieldColour} from '@blockly/field-colour';
```

Comment:
Note: this sample code is wrong. I need to import `registerFieldColour` as well.